.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+services:
+  - docker
+
+install:
+  - npm install -g dockerlint
+  - docker pull ubuntu:16.04
```

Comment:
Why? The [dockerfile](https://github.com/Vauxoo/docker-ubuntu-base/pull/11/files/ccb2477106df4410d3c15cb479b1ab3e97d126b3#diff-3254677a7917c6c01f55212f86c57fbfL1) in this PR is using 14.04